tests/x11/remote_desktop/screensharing_available.pm

```diff
@@ -12,7 +12,7 @@
 # - Check if wayland is being used
 #   - If wayland is detected, record_soft_failure (boo#1137569)
 # - Finish with ctrl-q
-# Maintainer: Pavel Dostál <pdostal@suse.cz>
+# Maintainer:
```

Comment:
this would be Desktop team, but qe-core can be it for now

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/actions/runs/4499546432/jobs/7917547747?pr=16691#step:6:83

Reply:
Do you have their e-mail and some github nickname to contact for approval?

Reply:
It is QE Core for now. You may merge.